CHANGES.rst

```diff
@@ -170,6 +170,7 @@ astropy.time
   These methods are similar to those in the Python standard library
   `time` package and provide flexible input and output formatting. [#7323]
 
+- Added `numpy.datetime64` format class for ``Time`` class. [#7361]
```

Comment:
Say
```
- Added ``datetime64`` format to the ``Time`` class to support working with 
  `numpy.datetime64` dtype arrays. [#7361]
```

Reply:
Docs build failing.

```
../CHANGES.rst:173: WARNING: py:obj reference target not found: numpy.datetime64
```

Reply:
I'll do 
```
``numpy.datetime64``
```

Reply:
Needs a twiddle: `~numpy.datetime64`.

Reply:
Sorry, in `CHANGES.rst`, the double back ticks are the way to go, so keep your change!